_posts/2018-08-31-grpc-packaging-releases-nightlies.md

```diff
@@ -0,0 +1,78 @@
+changed to `-pre1` indicating it is the first release candidate of that version (subsequent RCs, if any, will get `-pre2`, *etc.* suffixes.)
+
+A package build job is then triggered on the prerelease branch to create binary packages for the supported languages and platforms.  The binary packages undergo *distribution tests*, which are tests that verify correct installation and loading of the binary packages on supported platforms.  If everything looks fine as expected, the release manager then tags `HEAD` of the release branch as a "prerelease" on GitHub and the prerelease package builds will be uploaded to the common package managers of the supported languages.
+
+Mission-critical users depending on the latest version of gRPC in their application should run their continuous tests against the latest prerelease, as well as the latest release, to see possible regressions early and file issues and/or temporarily pin the dependency version to guard against breakages.
+
+After the branch is cut and the prerelease is evaluated in the wild
+for a week or two, the release candidate gets promoted to the final version.
+Bug fixes landing in the release branch after the first prerelease induce subsequent prerelease and qualification processes (albeit often with a shorter timescale) to decrease the risk of regressions the moment a
+new stable package is uploaded to the language package managers.
+Strictly adhering to this ritual is critical as even small bug fixes
+may have unexpected side-effects, wherefore this policy should only be
+short-circuited in extreme cases--*yes, we've learned it the hard way*.
+
+### Final Release ###
+
+Once a prerelease is qualified, it is promoted to the final version,
+ideally intact except for the version bump, removing the prerelease
+suffix, tagging the final minor version on GitHub, and shipping the
+final binaries on the common package managers.
+
+Subsequent bug fixes that warrant a patch release for that minor
+version trigger the prerelease and qualification process again.
+
+## Nightlies: Daily Package Builds ##
+
+gRPC supports many language and platform combinations.  While build
+systems like [Bazel](https://bazel.build) improve the build from source
+and distribution experience for scenarios where it is supported, setting
+up the infrastructure to build gRPC from source can be a hurdle for
+some people who like to experiement with the bleeding edge gRPC code
+on the `master` branch.  To alleviate this concern for the users who
+want to be on the bleeding edge, or just cannot wait for a particular
+bug fix or feature on `master` to land in an official release, as well
+as facilitating bisection while debugging by providing prebuilt binaries
+at regular intervals, [we produce packages](https://github.com/grpc/grpc/tree/master/tools/internal_ci/linux/grpc_publish_packages.sh) every day and upload them to
+a [Google Cloud Storage bucket](gs://packages.grpc.io), also
```

Comment:
bad link?

Reply:
It's not a bad link. It's a Google Cloud Storage bucket URI. Not loadable in the browser.